packages/extension-ui/package.json

```diff
@@ -31,13 +31,13 @@
     "@types/bn.js": "^4.11.6",
     "@types/enzyme": "^3.10.8",
     "@types/enzyme-adapter-react-16": "^1.0.6",
-    "@types/react-copy-to-clipboard": "^4.3.0",
+    "@types/react-copy-to-clipboard": "^5.0.0",
     "@types/react-dom": "^17.0.0",
-    "@types/react-router": "^5.1.8",
-    "@types/react-router-dom": "^5.1.6",
+    "@types/react-router": "^5.1.11",
+    "@types/react-router-dom": "^5.1.7",
     "@types/sinon-chrome": "^2.2.10",
-    "@types/styled-components": "^5.1.5",
-    "@wojtekmaj/enzyme-adapter-react-17": "^0.3.2",
+    "@types/styled-components": "^5.1.7",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.4.1",
```

Comment:
This reminds me - we just need to check every-so-often if the headline enzyme supports React 17 yet (this is a community fork of that to add support)

Reply:
Ah nice, and here's the link to the issue to keep an eye on https://github.com/enzymejs/enzyme/issues/2429